src/algo/LCS.js

```diff
@@ -426,6 +453,11 @@ export default class LCS extends Algorithm {
 		);
 		this.cmd(act.setForegroundColor, header, '#49AB40'); // sets text color
 
+		const lcsLength = this.nextIndex++;
+		this.oldIDs.push(lcsLength);
+		this.cmd(act.createLabel, lcsLength, '0', SEQUENCE_START_X + 180, SEQUENCE_START_Y - 25, 0);
```

Comment:
I like this! Honestly I'm a little surprised we didn't display the length of the LCS before...

My only thought is that it might be a little misleading to have the label say 
"Longest Common Subsequence: [length of LCS]", with the actual LCS below. 
We do sometimes ask explicitly for both the LCS and the length of the LCS on tests and I don't wanna accidentally add any confusion to those definitions.

Maybe we could add an additional explanatory label so that it reads
"Longest Common Subsequence: [LCS]"
"Length of LCS: [Length of LCS]"

Let me know what you think!

Reply:
Good point, I'll do that!